### PR TITLE
feat: add stable version 1.57 for Inovelli LZW31-SN

### PR DESCRIPTION
**Pull request by leojhartiv**
Added stable version 1.57 for Inovelli LZW31-SN

### CODE REVIEW

**Comment by stu1811** [comment]
Target 1 should be https://files.inovelli.com/firmware/LZW31-SN/1.57/LZW31-SN_1.45.bin

**Comment by leojhartiv** [comment]
> commented

Great catch!  Fixed!

File: firmwares/inovelli/LZW31-SN.json [hunk 1]
```diff
@@ -25,6 +25,23 @@
 					"integrity": "sha256:e07cb9972bfe88e143c72c9b6ed88a640b9d969fbf09d6c1a4625f711979b541"
 				}
 			]
+		},
+		{
+			"version": "1.57",
+			"channel": "stable",
+			"changelog": "Fixed - Optimization of Aux mode in neutral setting. – Fixed - The inoperative issue after changing parameter 8 from high value to low value.",
```
Comment (AlCalzone) [inline firmwares/inovelli/LZW31-SN.json:32]:
```suggestion
			"changelog": "Fixed - Optimization of Aux mode in neutral setting.\nFixed - The inoperative issue after changing parameter 8 from high value to low value.",
```
Not sure if HA's renders this as markdown currently, but we should at least attempt to have a line break here.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
Also, looks like the github action cannot post for forks, but the integrity hash is incorrect:

Error: **firmwares/inovelli/LZW31-SN.json**, version **1.57**, target **0**
Integrity hash mismatch
```diff
Expected: sha256:5c1d59906e302018c5c8bcd90040f0d545c8567a9c595a503df0ed27b77d5f33
Got:      sha256:c58d970e7148c000798e0bef3f214be4cbf93f62a784738e9b9c30b516b91e61
```

I guess this wasn't computed using the decoded HEX data:
> The integrity hash must be generated using the raw firmware data as it will be transferred to the device. If the firmware file is Intel HEX encoded or similar, this must be the hash of the decoded data. https://github.com/zwave-js/firmware-updates/blob/main/docs/firmware-files.md#defining-the-possible-upgrades-for-a-device

**Comment by leojhartiv** [comment]
> Also, looks like the github action cannot post for forks, but the integrity hash is incorrect:
> 
> Error: **firmwares/inovelli/LZW31-SN.json**, version **1.57**, target **0** Integrity hash mismatch
> 
> ```diff
> Expected: sha256:5c1d59906e302018c5c8bcd90040f0d545c8567a9c595a503df0ed27b77d5f33
> Got:      sha256:c58d970e7148c000798e0bef3f214be4cbf93f62a784738e9b9c30b516b91e61
> ```
> 
> I guess this wasn't computed using the decoded HEX data:
> 
> > The integrity hash must be generated using the raw firmware data as it will be transferred to the device. If the firmware file is Intel HEX encoded or similar, this must be the hash of the decoded data. https://github.com/zwave-js/firmware-updates/blob/main/docs/firmware-files.md#defining-the-possible-upgrades-for-a-device

Thanks for the help with this!  Couldn't figure out why I was hashing wrong!

**Comment by AlCalzone** [comment]
I've asked @InovelliUSA to check this, then we can merge.

**Comment by InovelliUSA** [comment]
Looks good to me. I didn't check the hash's but it looks like they were reviewed by the community.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**